styles/_base.scss

```diff
@@ -122,12 +122,31 @@ form {
   background-color: $emphasis1;
 }
 
-#pageContainer {
-  padding: 0 25px 25px;
+#pageNavContainer {
+  position: relative;
+  z-index: 2;
+}
+
+#contentArea {
+  position: relative;
+  z-index: 1;
+}
 
+#pageContainer {
+  position: relative;
   // temporary until we have a loading modal
   &:empty::after {
     content: "Loading...";
     font-size: 30px;
   }
 }
+
+#chatContainer {
+  position: fixed;
+  top: 100px;
+  right: 0;
+  width: $chatClosed;
+  .chatOpen & {
```

Comment:
somewhat minor and i think a linter would help use here, but can't we have a new line before a nested selector (unless it's the first item)?